packages/compass/src/main/application.ts

```diff
@@ -121,8 +122,8 @@ class CompassApplication {
     );
 
     // needs to happen after setupProtocolHandlers
-    if ((await import('electron-squirrel-startup')).default) {
-      debug('electron-squirrel-startup event handled sucessfully\n');
+    if (await handleSquirrelWindowsStartup()) {
+      app.quit();
```

Comment:
I really think it is much clearer what's going on when a library function isn't the one quitting the app. Reads better IMHO if that happens with the rest of the code that does that sort of thing. ie. this file.